### PR TITLE
Fix type for ovirt_auth

### PR DESCRIPTION
**Pull request by mscherer**
##### ISSUE TYPE
- Bugfix Pull Request
##### COMPONENT NAME

ovirt_auth
##### SUMMARY

Do not leak the password in log, and verify the path of ca_file

### CODE REVIEW

**Comment by gregdek** [comment]
maintainer unknown ping @gregdek @resmo @robynbergeron.

File: cloud/ovirt/ovirt_auth.py [hunk 1]
```diff
@@ -171,8 +171,8 @@ def main():
         argument_spec=dict(
             url=dict(default=None),
             username=dict(default=None),
-            password=dict(default=None),
-            ca_file=dict(default=None),
+            password=dict(default=None, no_log=True),
```
Comment (machacekondra) [inline cloud/ovirt/ovirt_auth.py:171]: There is a comment specifing to add no_log: true to ovirt_auth task. I wasn't aware that it's possible to pass it to specific argument. Can you please remove that comment from the example task and the no_log: true?

Thanks.

**Comment by machacekondra** [comment]
I've just propsed myself as maintainer of oVirt modules:

 https://github.com/ansible/ansibullbot/pull/181

**Comment by gregdek** [comment]
Thanks @mscherer. To the current maintainers, @machacekondra please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/ovirt/ovirt_auth.py [hunk 2]
```diff
@@ -98,8 +98,6 @@
        # oVirt user's password, and include that yaml file with variable:
        - include_vars: ovirt_password.yml
 
-       # Always be sure to pass 'no_log: true' to ovirt_auth task,
-       # so the oVirt user's password is not logged:
        - name: Obtain SSO token with using username/password credentials:
          no_log: true
```
Comment (machacekondra) [inline cloud/ovirt/ovirt_auth.py]: ^ This line should be removed as well, but not important.

**Comment by machacekondra** [comment]
shipit

**Comment by gregdek** [comment]
Thanks again to @mscherer for this PR, and thanks @machacekondra for reviewing. Marking for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by resmo** [comment]
@mscherer the maintainer did an inline comment about the doc, which should be also updated. I wait with the merge so you can fix it too. ;)

**Comment by abadger** [comment]
Also cherry-picked this to stable-2.2.

**Pull request merged**